adminactions/export.py

```diff
@@ -55,7 +56,8 @@ def base_export(modeladmin, request, queryset, title, impl, name, template, form
                'select_across': request.POST.get('select_across') == '1',
                'action': get_action(request),
                'columns': [x for x, v in cols]}
-    # initial.update(csv_options_default)
+    if initial["action"] == "export_as_csv":
+        initial.update(getattr(settings, "CSV_OPTIONS_DEFAULT", {}))
```

Comment:
@scrck can we perhaps add an `ADMINACTIONS_` prefix? We can name the setting `ADMINACTIONS_CSV_EXPORT_DEFAULT`, even though it's quite long ...

Alternatively, we could add an `ADMINACTIONS_CONFIG` dictionary and include any customization there; what do you think? We could then even use generic naming, e.g:

```python
CONFIG = getattr(django.conf.settings, 'ADMINACTIONS_CONFIG', {})

def base_export(modeladmin, request, queryset, title, impl, name, template, form_class, ):
    # ...
    action_form_settings = '%s_FORM_DEFAULTS' % initial["action"].upper()
    if action_form_settings in CONFIG:
        initial.update(CONFIG[action_form_settings])
```